schemas/nl.basjes/yauaa_context/jsonschema/1-0-4

```diff
@@ -0,0 +1,234 @@
+{
+    "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+    "description": "Schema for an entity context generated by the YAUAA enrichment after parsing the user agent",
+    "self": {
+        "vendor": "nl.basjes",
+        "name": "yauaa_context",
+        "format": "jsonschema",
+        "version": "1-0-4"
+    },
+    "type": "object",
+    "properties": {
+        "deviceClass": {
+            "description": "See https://yauaa.basjes.nl/README-Output.html",
+            "enum": ["Desktop", "Anonymized", "Unknown", "UNKNOWN", "Mobile", "Tablet", "Phone", "Watch", "Virtual Reality", "eReader", "Set-top box", "TV", "Game Console", "Home Appliance", "Handheld Game Console", "Voice", "Car", "Robot", "Robot Mobile", "Spy", "Hacker", "Augmented Reality", "Robot Imitator"]
+        },
+        "deviceName": {
+            "description": "Example: Google Nexus 6",
+            "type": ["string", "null"],
+            "maxLength": 256
+        },
+        "deviceBrand": {
+            "description": "Example: Google",
+            "type": ["string", "null"],
+            "maxLength": 128
+        },
+        "deviceCpu": {
+            "type": ["string", "null"],
+            "maxLength": 128
+        },
+        "deviceCpuBits": {
+            "type": ["string", "null"],
+            "maxLength": 128
+        },
+        "deviceFirmwareVersion": {
+            "type": ["string", "null"],
+            "maxLength": 1000
+        },
+        "deviceVersion": {
+            "type": ["string", "null"],
+            "maxLength": 1000
+        },
+        "operatingSystemClass": {
+            "description": "See https://yauaa.basjes.nl/README-Output.html",
+            "type": ["string", "null"],
```

Comment:
You added "null" as a type... but this does not make any difference, unless you also add `null` to the enum array.  As currently written, null is still not an allowed value for `operatingSystemClass`.

Reply:
Yes, I did not realise that. I think if we keep null as an allowed type, I'll add it to the enum.